record/record.app.js

```diff
@@ -112,7 +112,8 @@
 
                 $rootScope.tableModels = [];
                 $rootScope.lastRendered = null;
-
+                var cutOff = (angular.isUndefined(chaiseConfig.maxRelatedTab) || chaiseConfig.maxRelatedTab === null || chaiseConfig.maxRelatedTab == " ")?Infinity:chaiseConfig.maxRelatedTab;
```

Comment:
This conditional seems overly expressive. You could verify this by checking `maxRelatedTab > 0 ? maxRelatedTab : Infinity`.